fe/fe-core/src/main/java/com/starrocks/qe/scheduler/Deployer.java

```diff
@@ -52,6 +57,9 @@
  */
 public class Deployer {
     private static final Logger LOG = LogManager.getLogger(Deployer.class);
+    private static final ThreadPoolExecutor EXECUTOR =
+            ThreadPoolManager.newDaemonCacheThreadPool(ThreadPoolManager.cpuCores(),
```

Comment:
Is it better to use a new configuraion to control the thread pool's size?

Reply:
The thread pool size is similar to ForkJoinPool's, but pending tasks will be queued in `EXECUTOR`, I think it is good enough.